src/medicine/main.py

```diff
@@ -34,8 +34,15 @@ def read_pickle_file(file_name):
 
 
 # ic(Medicine.instances)
-data = read_pickle_file('instances.p')
-neurotop_600.remove_instance_from_file()
-ic(read_pickle_file('instances.p'))
 
-print(data['lacosamide_100'][1])
\ No newline at end of file
+
+def run():
+    data = read_pickle_file('instances.p')
+    neurotop_600.remove_instance_from_file()
+    ic(read_pickle_file('instances.p'))
+
+    print(data['lacosamide_100'][1])
+
+
+if __name__ == '__main__':
```

Comment:
to oznacza że jeżeli ten plik został wprost odpalony, typu `python src/medicine/main.py`, to wykonaj ten kod. Jest to dość istotne, bo inaczej jak byś chciał zaimportować coś z tego pliku (a zawsze zakładaj że chcesz, chociażby dla testów), to najpierw wywołuje plik (w dużym uproszczeniu). W tym przypadku to nic złego się nie stanie, bo tylko coś wypisze, ale jakbyś np. miał coś co wysyła wiadomości albo coś, to może być większym problemem